File: Datasets/HumanEval/HumanEval_Java/26_all.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class RemoveDuplicates {
    /**
     * From a list of integers, remove all elements that occur more than once.
     * Keep order of elements left the same as in the input.
     * >>> remove_duplicates([1, 2, 3, 2, 4])
     * [1, 3, 4]
     * >>> remove_duplicates([1, 1,1, 2, 2, 4])
     * [4]
     * @param numbers a list of integers
     * @return the input list without the elements that occur more than once
     */
    public static List<Integer> removeDuplicates(List<Integer> numbers) {

File: Cleared_Dataset/HumanEval_Py/26_all.py
from typing import List


def remove_duplicates(numbers: List[int]) -> List[int]:
    """ From a list of integers, remove all elements that occur more than once.
    Keep order of elements left the same as in the input.

    args:
        numbers: list of integers
    
    returns:
        list of integers with all elements that occur more than once removed

    examples:
    >>> remove_duplicates([1, 2, 3, 2, 4])
    [1, 3, 4]
    >>> remove_duplicates([7, 5, 6, 5, 7, 3])
    [6, 3]
    """
